content/Software Support/Installation/Add-libraries-to-Arduino-IDE.md

```diff
@@ -67,7 +59,7 @@ Learn how to add libraries to Arduino IDE.
 
 ---
 
-## Using installed libraries
+## Using installed libraries {#using-libraries}
 
 When the library has been installed you can do the following:
 
```

Comment:
The last sentence in this bullet point: "* Open any included example sketches from _File > Examples > \<Library Name\>_ in the top menu bar. Note that all libraries do not come with examples." can be improved to something like: "Note that not all libraries come with examples."